### DISCUSSION TITLE
Can I build a sample client on the mac?

### DISCUSSION THREAD

**Discussion by kimcoldwater** [FreeRDP/FreeRDP#8109] [Q&A]
hello, 

I built and customized the sample client on Linux.

I want to use it in the Mac to develop and test it more comfortably.

I added a sample client folder to the mac part in CmakeLists.txt.

-DWITH_SAMPLE=ON I tried to build it by inserting the option and fixing the error, but it failed.

Can you advise me if there is a way to build a sample client on the mac?

Thank you for reading the question.

**Comment by akallabeth** [comment]
compiles just like on linux, so it is most likely you´re doing something different?
